src/content/blog/2025-09-05-memorable-wallet-addresses-custom-domain.md

```diff
@@ -0,0 +1,248 @@
+
+Note that, given the lack of control over headers, you may face CORS issues as explained above, but it'll work with most other Open Payment uses.
+
+More importantly, you will have to ensure the content of your `pay.json` matches the JSON response that was received from your original wallet address, in case it changes in future.
+
+## How does it work with Web Monetization
+
+### The `<link>` element
+
+A payment pointer, such as `$mywebsite.com`, is convenient for many purposes. However, when you want to add your wallet address to your website as a Web Monetization receiver, you need to convert that payment pointer to the Open Payments "wallet address" format for use with the `<link>` tag ([why?](https://github.com/WICG/webmonetization/issues/19#issuecomment-705407129)). Slightly disappointing, but hey, you only need to do this once!
+
+You can use the [link tag generator](https://webmonetization.org/tools/link-tag/) to convert the payment pointer to a valid link tag.
+
+If you're using the same domain as your website, you can use the link element's URL resolution by writing the `<link>` tag like following:
+
+```html
+<link rel="monetization" href="/.well-known/pay" />
+```
+
+Aside: During local development (i.e. with localhost or custom dev domain), if you have the Web Monetization browser extension installed and are using a CDN or host-level configuration, the extension won't resolve to your actual wallet address, and no real money will be sent during regular website development, which can be very handy. And when you want to actually test Web Monetization integrations, you can resolve the URL to a different [test wallet](https://wallet.interledger-test.dev/) address!
+
+### Usage in extension
+
+The Web Monetization browser extension allows you to use any payment pointer or wallet address to connect your wallet as a sender. This means using a custom, branded payment pointer works the same as using your wallet's address directly. So, feel free to use your branded payment pointer there as well!
+
+Once you've connected using your custom payment pointer, the extension will show you both the custom payment pointer and the corresponding original wallet address on the Settings screen.
+
+Aside: The Web Monetization API doesn't allow the websites to know what payment pointer/wallet address was used by the payer, so they cannot directly correlate the payments to your identity (your domain name). The website's wallet may display information about the sender, but it'll only show the original wallet address there, not your custom alias.
+
+### `MonetizationEvent`
+
+Let's say, instead of using a wallet address that our provider gave us, we alias it to a custom address as shown above, and add it to our webpage. This sort of indirection can also arise from using the [probabilistic revenue sharing generator](https://webmonetization.org/tools/prob-revshare/), or from your wallet provider may be using an aliased wallet address itself.
```

Comment:
> from your wallet provider may be using

Missing something here

Reply:
Made some changes in https://github.com/interledger/interledger.org-developers/pull/151/commits/cd62d323334b9bd4f1bcfe3aa9ae4c35b89ffde7 and https://github.com/interledger/interledger.org-developers/pull/151/commits/4ec07659df7c9ab274f4e88f2820c0866ea60d19. Let me know if it's better.